Dockerfile

```
@@ -2,10 +2,15 @@ FROM thmhoag/steamcmd:latest
 
 USER root
 
-RUN apt-get update && \
-    apt-get install -y curl cron bzip2 perl-modules lsof libc6-i386 lib32gcc1 sudo
+RUN apt-get update \
+    && apt-get install -y curl cron bzip2 perl-modules lsof libc6-i386 lib32gcc1 sudo \
+    && apt-get autoremove -y \
+    && apt-get clean -y \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /tmp/* \
+    && rm -rf /var/tmp/*
 
-RUN curl -sL "https://raw.githubusercontent.com/FezVrasta/ark-server-tools/v1.6.54/netinstall.sh" | bash -s steam && \
+RUN curl -sL "https://raw.githubusercontent.com/arkmanager/ark-server-tools/v1.6.57/netinstall.sh" | bash -s steam && \
```

```suggestion
RUN curl -sL https://git.io/arkmanager | bash -s steam && \
```
maybe we could follow the documentation to install the last version

---

Thanks for reviewing @edausq, makes totally sense for me.